Doc/library/string.rst

```diff
@@ -588,6 +589,14 @@ The available presentation types for :class:`float` and
    |         | as altered by the other format modifiers.                |
    +---------+----------------------------------------------------------+
 
+The available presentation types for :class:`complex` are same as for
+:class:`float`.  Both real and imaginary components of a complex number are
+formatted as floating-point numbers, accordingly specified presentaion type.
+They separated by mandatory sign of the imaginary part, terminated by ``j``
+suffix.  If presentation type is missing --- result match the output of
+:func:`str` (numbers also surrounded by round parentheses, if real part is
```

Comment:
Ok, lets keep your suggestion:
```suggestion
:func:`str` (numbers also surrounded by parentheses, if real part is
```
This pr was approved, so I'll not commit it.

FYI, current wording was copied from the complex type docs: https://docs.python.org/3.14/library/functions.html#complex

Reply:
I think removing "round" is a good change.

Reply:
Ah, @skirpichev, indeed -- sorry for dropping that unnecessary explainer on you!